src/server/game/Server/Packets/LFGPackets.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "LFGPackets.h"
+
+void WorldPackets::LFG::LFGJoin::Read()
+{
+    _worldPacket >> Roles;
+    _worldPacket.read_skip<uint16>();
+    _worldPacket >> Slots;
+    for (int8 i = 0; i < Slots; ++i)
+        _worldPacket >> dungeonId;
```

Comment:
This is broken anyways

Reply:
>This is utterly broken anyways

The message above is broken too because it doesn't explain in any way which part is broke, why, how it should be instead or how to fix it.